diff --git a/scripts/buildorder.py b/scripts/buildorder.py
--- a/scripts/buildorder.py
+++ b/scripts/buildorder.py
@@ -10,0 +11,4 @@
+codev_excluded_subpackages = set(filter(
+    None,
+    (os.getenv('CODEV_EXCLUDED_SUBPACKAGES') or '').split(','),
+))
@@ -162 +166,4 @@
-                if f"{self.name}-static" != subpkg.name:
+                if (
+                    f"{self.name}-static" != subpkg.name
+                    and subpkg.name not in codev_excluded_subpackages
+                ):
diff --git a/packages/python/build.sh b/packages/python/build.sh
--- a/packages/python/build.sh
+++ b/packages/python/build.sh
@@ -20 +19,0 @@
-TERMUX_PKG_BUILD_DEPENDS="tk"
@@ -22 +20,0 @@
-TERMUX_PKG_SUGGESTS="python-tkinter"
@@ -154 +152 @@
-	for module in _bz2 _curses _lzma _multiprocessing _sqlite3 _ssl _tkinter zlib _zstd; do
+	for module in _bz2 _curses _lzma _multiprocessing _sqlite3 _ssl zlib _zstd; do
